wc-calypso-bridge-class.php

```diff
@@ -89,7 +89,15 @@ public function includes() {
 		 * See https://github.com/Automattic/jetpack/pull/8296 for more details
 		 */
 		if ( ! class_exists( 'Jetpack_WooCommerce_Analytics' ) ) {
```

Comment:
Should we ensure the class_exists( 'Jetpack' ) too? Granted it should always be loaded on AT sites, might not hurt to add the check in this repo.

Reply:
Check added in https://github.com/Automattic/wc-calypso-bridge/pull/20/commits/020adb2dfb1e6d49e98a83da3fe6314253b54c9a